src/cache/precomputed_quantities.jl

```diff
@@ -122,9 +122,6 @@ function precomputed_quantities(Y, atmos)
             ᶜentrʲs = similar(Y.c, NTuple{n, FT}),
             ᶜdetrʲs = similar(Y.c, NTuple{n, FT}),
             ᶠnh_pressure³ʲs = similar(Y.f, NTuple{n, CT3{FT}}),
-            ᶜS_q_totʲs = similar(Y.c, NTuple{n, FT}),
-            ᶜS_q_tot⁰ = similar(Y.c, FT),
```

Comment:
You still need these two?

Reply:
https://github.com/CliMA/ClimaAtmos.jl/blob/40e6a2fa0a77aaa74d4ad666eb5f0d97711fef80/src/cache/precomputed_quantities.jl#L134

I am adding them here. But only for the 0M scheme. I guess diagnostic edmf doesn't really do dispatch...

Reply:
Ah, I see. It should be easy to add dispatch if needed.

Reply:
I might have to add another loop over levels for the precipitation dispatch. But maybe that would be ok? I'll think about it